client/src/components/Card/Card.js

```diff
@@ -8,7 +8,7 @@ const Card = styled.div`
   background: ${(props) => (props.primary ? "var(--primary-color)" : "white")};
   border-radius: 15px;
   width: 100%;
-  max-width: 500px;
+  /* max-width: 500px; */
```

Comment:
No commented out code on main :) You can always check back in your git history if you want to reuse previous code